CHANGELOG.md

```diff
@@ -54,10 +54,17 @@ In order not to break all angular panel plugins and data sources we have some cu
 
 # 7.3.5 (2020-12-10)
 
+### Features and enhancements
+
+* **Alerting**: Improve Prometheus Alert Rule error message. [#29390](https://github.com/grafana/grafana/pull/29390), [@wbrowne](https://github.com/wbrowne)
+
 ### Bug fixes
 
+* **Alerting**: Fix alarm message formatting in Dingding. [#29482](https://github.com/grafana/grafana/pull/29482), [@tomowang](https://github.com/tomowang)
 * **AzureMonitor**: Fix unit translation for MilliSeconds. [#29399](https://github.com/grafana/grafana/pull/29399), [@secustor](https://github.com/secustor)
 * **Instrumentation**: Fix bug with invalid handler label value for HTTP request metrics. [#29529](https://github.com/grafana/grafana/pull/29529), [@bergquist](https://github.com/bergquist)
+* **Prometheus**: Fix display name field has no effect. [#29441](https://github.com/grafana/grafana/pull/29441), [@zoltanbedi](https://github.com/zoltanbedi)
```

Comment:
Not sure what "Fix display name field has no effect." means. Can this be a little more clear? Thank you.

Reply:
@zoltanbedi

Reply:
I really have no idea what else could be in the there to be more clear. Any suggestions?

Reply:
updated PR, with slightly more descriptive title: "Prometheus: Fixes problem where changing display name in Field tab had no effect"

Reply:
```suggestion
* **Prometheus**: Fix bug where display name field has no effect. [#29441](https://github.com/grafana/grafana/pull/29441), [@zoltanbedi](https://github.com/zoltanbedi)
```

Reply:
Or this
```suggestion
* **Prometheus**: Fix issue where display name field has no effect. [#29441](https://github.com/grafana/grafana/pull/29441), [@zoltanbedi](https://github.com/zoltanbedi)
```

Reply:
How about "Fix issue where the changing the field display name was not having any effect". Hope I understood the issue (https://github.com/grafana/grafana/issues/29437) correctly and this helps.